eng/tools/suppressions/package-lock.json

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
FYI I am unsure why this `package-lock.json` was missing. I added it because I wanted to `npm run test` from the `suppressions` directory.

Reply:
The standard workflow in `azure-rest-api-specs` is to run `npm ci` in the repo root to install deps for all folders in the repo.  This should allow you to exec `npm run test` under any folder under `eng`.

It's fine if you want to work locally on a subset of the repo, but I'd like to keep the repo itself as clean as possible (so only a single root package-lock.json).